src/ign.hh

```diff
@@ -34,7 +34,8 @@ extern "C" SDFORMAT_VISIBLE int cmdCheck(const char *_path);
 
 /// \brief External hook to read the library version.
 /// \return C-string representing the version. Ex.: 0.1.2
-extern "C" SDFORMAT_VISIBLE char *ignitionVersion();
+extern "C" SDFORMAT_VISIBLE char SDF_DEPRECATED(13) *ignitionVersion();
```

Comment:
This header isn't installed, so we don't need to tick-tock. Just changing to `gz` is enough.

Reply:
https://github.com/gazebosim/sdformat/pull/1040/commits/ac77e5ea225ce755a66357d1ed2bff935bd789f5 !